packages/documentation/docs/migration/2_0_0.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 0
+sidebar_title: Migration to 2.0.0
+title: Migration to 2.0.0
+---
+
+# ix-menu
+
+## Item dropdown overflow is removed
+
+We replaced the overflow menu with second level navigation in form of categories
+
+```tsx
+<ix-menu-category label="AI Configuration" icon="rocket">
+  <ix-menu-item>Nested Item 1</ix-menu-item>
+  <ix-menu-item>Nested Item 2</ix-menu-item>
+</ix-menu-category>
+```
+
+If the menu stills overflow the bottom of the menu a scroll container will be displayed.
+
+## Multiple screen layouts
+
+The `ix-basic-navigation` in combination of the `ix-menu` comes with 3 layout formats:
```

Comment:
It is not clear to the reader what was changed. We can re-write this section together :)